tools/fog-local-network/local_fog.py

```diff
@@ -31,6 +31,13 @@
 IAS_SPID = os.getenv('IAS_SPID', default='0'*32) # 16 bytes
 
 FOG_SQL_DATABASE_NAME = 'fog_local'
+# Use env.DATABASE_URL if it exists, else use postgres://$PGHOST/fog_local,
+# falling back to postgres://localhost/fog_local.
+DATABASE_URL_ENV = 'DATABASE_URL=${DATABASE_URL:-postgres://${PGHOST:-localhost}/%s}' % FOG_SQL_DATABASE_NAME
```

Comment:
Not sure what this is doing here

Reply:
IDK either, maybe it's not needed? I guess the cherry-pick procedure decided to pull it in T_T